### PR TITLE
networking-operators-redirects: Created redirects for AWS Operator docs

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16+

Issue:
Redirects for https://github.com/openshift/openshift-docs/pull/84683

### CODE REVIEW

**Comment by bscott-rh** [comment]
redirects LGTM after rebase/tests are resolved

File: .s2i/httpd-cfg/01-commercial.conf [hunk 1]
```diff
@@ -175,15 +175,50 @@ AddType text/vtt                            vtt
...
     RewriteRule ^container-platform/4\.10/networking/external_dns_operator/nw-installing-external-dns-operator.html /container-platform/4.10/networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.html [NE,R=301]
 
-   # Redirect for renamed Ingress Controller document
+    # Redirect for renamed Ingress Controller document
     RewriteRule ^container-platform/(4\.1[2-6])/networking/nw-ingress-controller-endpoint-publishing-strategies\.html$ /container-platform/$1/networking/nw-configuring-ingress-controller-endpoint-publishing-strategy.html [NE,R=302,L]
 
     # Redirect for Networking operators move per https://github.com/openshift/openshift-docs/pull/85085
     RewriteRule ^container-platform/(4\.1[6-8])/networking/aws_load_balancer_operator/(.*)? /container-platform/$1/networking/networking_operators/aws_load_balancer_operator/$2 [NE,R=302]
+
+    RewriteRule rosa/networking/aws-load-balancer-operator.html /container-platform/$1/networking/networking_operators/aws-load-balancer-operator.html [NE,R=302]
```
Comment (dfitzmau) [inline .s2i/httpd-cfg/01-commercial.conf:184]: Hi @bscott-rh . Is this syntax correct? Rosa redirects are also needed :grimacing: .

File: .s2i/httpd-cfg/01-commercial.conf [hunk 2]
```diff
@@ -175,15 +175,50 @@ AddType text/vtt                            vtt
...
+    # Redirect for the Installing the MetalLB Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/metallb/metallb-operator-install.html /container-platform/$1/networking/networking_operators/metallb-operator/metallb-operator-install.html [NE,R=302]
+    # Redirect for the Upgrading the MetalLB Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/metallb/metallb-upgrading-operator.html /container-platform/$1/networking/networking_operators/metallb-operator/metallb-upgrading-operator.html [NE,R=302]
+    # Redirect for the Cluster Network Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/cluster-network-operator.html /container-platform/$1/networking/networking_operators/cluster-network-operator.html [NE,R=302]
+    # Redirect for the DNS Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/dns-operator.html /container-platform/$1/networking/networking_operators/dns-operator.html [NE,R=302]
+    # ROSA/Dedicated Redirect for the DNS Operator document
```
Comment (dfitzmau) [inline .s2i/httpd-cfg/01-commercial.conf:203]: Hi @bscott-rh . Is this syntax correct? Rosa and Dedicated redirects are also needed :grimacing:

File: .s2i/httpd-cfg/01-commercial.conf [hunk 3]
```diff
@@ -175,15 +175,50 @@ AddType text/vtt                            vtt
...
+    RewriteRule ^(rosa|dedicated)networking/dns-operator.html /$1/networking/networking_operators/dns-operator.html [NE,R=302]
```
Comment (bscott-rh) [inline .s2i/httpd-cfg/01-commercial.conf:204]:
```suggestion
    RewriteRule ^(rosa|dedicated)/networking/dns-operator.html /$1/networking/networking_operators/dns-operator.html [NE,R=302]
```

**Comment by bscott-rh** [comment]
1 small typo, otherwise redirects LGTM.

**Pull request merged**